src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String[] splitDateFrom = dateFrom.split("[.]");
+        String[] splitDateTo = dateTo.split("[.]");
+        int datFrom = Integer.parseInt(splitDateFrom[0]);
+        int monthFrom = Integer.parseInt(splitDateFrom[1]);
+        int yearFrom = Integer.parseInt(splitDateFrom[2]);
+        int datTo = Integer.parseInt(splitDateTo[0]);
+        int monthTo = Integer.parseInt(splitDateTo[1]);
+        int yearTo = Integer.parseInt(splitDateTo[2]);
+        LocalDate from = LocalDate.of(yearFrom, monthFrom, datFrom);
+        LocalDate to = LocalDate.of(yearTo, monthTo, datTo);
```

Comment:
Let's use DateTimeFormatter to parse dates (don't forget: If you create a formatter, make it a constant)